cloudferrylib/os/image/glance_image.py

```diff
@@ -227,8 +227,8 @@ def delete_image(self, image_id):
         self.glance_client.images.update(image_id, protected=False)
         self.glance_client.images.delete(image_id)
 
-    def get_image_by_id(self, image_id):
-        for glance_image in self.get_image_list():
+    def get_image_by_id(self, image_id, is_filtering=True):
+        for glance_image in self.get_image_list(is_filtering):
```

Comment:
get-function returned not full information.(example `container_format`)
`{u'1a52c0b4-cddd-47f0-865b-2efb90171db3': {'image': {'owner_name': u'admin', 'resource': <cloudferrylib.os.image.glance_image.GlanceImage object at 0x7f3a9ffabf90>, u'name': u'image1', u'deleted': False, u'checksum': u'133eae9fb1c98f45894a4e60d8736619', u'min_ram': 0, 'properties': {}, u'owner': u'cb0ed10e99b8433fb546404bc1b99660', u'protected': False, 'members': {}, u'min_disk': 0, u'is_public': True, u'id': u'1a52c0b4-cddd-47f0-865b-2efb90171db3', u'size': 13200896}, 'meta': {'img_loc': 'file:///var/lib/glance/images/1a52c0b4-cddd-47f0-865b-2efb90171db3'}}}
`